src/service/position-observer-impl.js

```diff
@@ -92,13 +124,32 @@ class AbstractPositionObserver {
           Math.floor(Math.random() * LOW_FIDELITY_FRAME_COUNT) : 0,
       trigger(position) {
         const prePos = entry.position;
+
         if (prePos
             && layoutRectEquals(prePos.positionRect, position.positionRect)
             && layoutRectEquals(prePos.viewportRect, position.viewportRect)) {
           // position doesn't change, do nothing.
           return;
         }
+
+        // Add the relative position regardless of whether the element
+        // is inside the view or not
+        position.relativePos = layoutRectsRelativePos(
+            position.positionRect, position.viewportRect
+        );
+
+        // Determine if the element is fully, partially or not contained
+        // within the viewport
+        position.visibility = layoutRectsIntersectionState(
```

Comment:
I don't think we should include this either, consumer of the API can calculate both `visibility` and `visibleHeight` themselves.